src/mysql/row.rs

```diff
@@ -1,9 +1,15 @@
-use diesel::backend::Backend;
-use diesel::mysql::data_types::{MysqlTime, MysqlTimestampType};
-use diesel::mysql::{Mysql, MysqlType, MysqlValue};
-use diesel::row::{PartialRow, RowIndex, RowSealed};
-use mysql_async::consts::{ColumnFlags, ColumnType};
-use mysql_async::{Column, Row, Value};
+use diesel::{
+    backend::Backend,
+    mysql::{
+        data_types::{MysqlTime, MysqlTimestampType},
+        Mysql, MysqlType, MysqlValue,
+    },
+    row::{PartialRow, RowIndex, RowSealed},
+};
+use mysql_async::{
+    consts::{ColumnFlags, ColumnType},
+    Column, Row, Value,
+};
```

Comment:
I would prefer not to reformat all imports. Could you roll back all import formatting related changes?

Reply:
Sure, please checkout the latest changes.